src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -549,3 +549,7 @@
     (:less-than-three-minutes constants/wallet-transaction-estimation) "1-3"
     (:less-than-five-minutes constants/wallet-transaction-estimation)  "3-5"
     ">5"))
+
+(defn multi-account?
```

Comment:
This function is quite confusing. It is not clear what structure should be passed to it. 
Also it is very basic and used only in one place, so I'd suggest to not introduce it at all

Reply:
done